cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -1053,6 +1053,10 @@ spec:
                 configMapKeyRef:
                   name: {{ template "cost-analyzer.fullname" . }}
                   key: kubecost-token
+            {{- if and (.Values.kubecostAggregator) (gt (.Values.kubecostAggregator.replicas | toString | atoi) 0) }}
+            - name: WATERFOWL_ENABLED
```

Comment:
we still need these since the underlying code has not had renaming work done yet. Thinking was that this can be changed once that work is done without affecting how people are using the helm charts

Reply:
I think that would probably be fine, but is that planned before the next minor release?

Reply:
not sure on timeline. Goal with this change is to give eng leadership maximum flexibility for setting the priority of this